src/tyr/serializers.cc

```diff
@@ -223,9 +223,13 @@ void serializeIncidentProperties(rapidjson::Writer<rapidjson::StringBuffer>& wri
     writer.String(std::string(valhalla::incidentTypeToString(incident_metadata.type())));
   }
   if (!incident_metadata.iso_3166_1_alpha2().empty()) {
-    writer.Key(key_prefix + "iso_3166_1_alpha2");
+    writer.Key(key_prefix + "iso_3166_1");
```

Comment:
Renamed this to be consistent with attribute names in the `admins` object of the serialized response.

Reply:
This is a potentially api-breaking change though.

Reply:
Yes. I'm open to reverting it to its original name. However it'd be nice to have a consistent name for the ISO 2-char country code everywhere in the response. @kevinkreiser do you know of existing consumers of valhalla API that are relying on this name?

Reply:
i dont know anyone else that uses it but it is in a release already so its kind of evil. why not just leave it and have the alpha3 one have the suffix?

Reply:
I'll do that. When looking at the code initially, I found that we used a different name for iso2 code in the "admins" object.